.changeset/shy-actors-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@makeswift/runtime': patch
```

Comment:
Patch or minor?

Reply:
This is technically a breaking change. But AFAIK this only affects people that already use `@layer` and they rely on the reset CSS to override something.